### ISSUE TITLE
Feature: Disabling wishlists

### ISSUE THREAD

**Issue by idiosyn** [Intracto/SecretSanta#475]
I know this was already discussed in https://github.com/Intracto/SecretSanta/issues/162 and I am sorry for bringing this up again, but I would like to know if it is open to a reconsideration.

My request and situation are similar to @ShadowXVII's.

> At our organization, we don't want to use Wishlists at all: we don't want anyone even filling them out thinking that anyone will read them, and definitely don't want reminders sent (not even one) asking to fill out a wishlist.

> Wishlists should be an option at the Party level, so that it can be disabled and not used at all. The emails would not mention it, nor would participants be notified

> [...]

> We just plain don't do wishlists. It confuses participants when they get asked to fill one out.

@tvlooy said:

> Sometimes people ask for a specific feature that we don't have and won't do. If we have to add them all, our project gets bloated. The power of our solution is that it is simple and doesn't have too much knobs and buttons

If simplicity is one goal, I'd argue that an option to disable wishlists could make things much simpler for many groups. @ShadowXVII is right: wishlists (and the emails) often **confuse** people instead of helping them. I think this has been underestimated here.

It would be great if the party manager could simply disable wishlists. It would be an additional setting for the party manager only, who is usually more tech-savvy than other people in the group. If the cost of making things simpler for the majority of the people is one additional setting for one person only (the party manager), I think it's worthy.

I don't want to use wishlists, and currently I have to keep an eye on them (to make sure nobody is using them). I also need to check with everybody if they understood the wishlists are not supposed to be used, despite the email they received about it.

**I** know it's possible for them to unsubscribe, but **they** don't know it. And most of them will know it only after receiving the wishlist email. And when they receive it, it's too late, they are already confused. Most of them look only at the big red button, they simply ignore the unsubscribe link.

I love your work and I hope you can reconsider adding the option to disable wishlists.

Thank in advance.

**Comment by tvlooy** [comment]
If you can provide the code we will probably reconsider the feature.
